app/modules/individuals/resources.py

```diff
@@ -217,11 +217,58 @@ def patch(self, args, individual):
         context = api.commit_or_abort(
             db.session, default_error_message='Failed to update Individual details.'
         )
+
         with context:
             parameters.PatchIndividualDetailsParameters.perform_patch(
                 args, obj=individual
             )
-            db.session.merge(individual)
+
+        # copycat the pattern established in sightings module
+        edm_count = 0
+        for arg in args:
+            if (
+                'path' in arg
+                and arg['path'][1:]
+                in parameters.PatchIndividualDetailsParameters.PATH_CHOICES_EDM
+            ):
+                edm_count += 1
+        if edm_count > 0 and edm_count != len(args):
+            log.error(f'Mixed edm/houston patch called with args {args}')
+            abort(
+                success=False,
+                passed_message='Cannot mix EDM patch paths and houston patch paths',
```

Comment:
obvious question... Why not?

Reply:
I was following the pattern in encounters and sightings API. I suppose we could split a list into two, one with EDM args and one with Houston and send them down their various paths. It would make sense to do this in all three places to keep behavior consistency. I'm not opposed to it, but not champing at the bit to do it since it's not currently a blocker for anything.